### PR TITLE
add GetReward tx parse for Cosmos

### PR DESCRIPTION
**Pull request by Pantani**
closes https://github.com/trustwallet/blockatlas/issues/617

http://localhost:8420/v1/cosmos/cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46

```
{
  "total": 25,
  "docs": [
    {
      "id": "DA0B6F5328D6EE9DEC1A9F33F79397481877AD8963109FFC839F38A3DDA23484",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576465035,
      "block": 54872,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "422"
      }
    },
    {
      "id": "7571946B8C20EC561637B8CB32F35A8745DFD9184C233EDE193F278BDBFFD9AD",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464972,
      "block": 54863,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "422"
      }
    },
    {
      "id": "8BD8D28D20E7D6DFF83248F6D5D28A092627B2EFF87084BC14E700BC0E724961",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464909,
      "block": 54854,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "529"
      }
    },
    {
      "id": "10CA24F42D5FF3BE5FE3A3FE986E6663C9C5FB6BDDEEA5887AD089A545C04214",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464845,
      "block": 54845,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "328"
      }
    },
    {
      "id": "002279B9CB29EBBF7FCE8E76C867160E9D8A7D212CDE6DF6C74EAA9748EFF178",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464797,
      "block": 54838,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "375"
      }
    },
    {
      "id": "F1E1A89A1F721EE829F76B1553FEE7911C21506D2724409C7E63561D6C1A8FCC",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464740,
      "block": 54830,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "516"
      }
    },
    {
      "id": "66589F818C4988EC566F487E2996C0872207D48CE0FAC6A221A9AD3F341390AE",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464664,
      "block": 54819,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "905"
      }
    },
    {
      "id": "FB5EDA6CFA52C519D6CE6420DB77530147075933551472FE42751CE2ADC4F524",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464547,
      "block": 54802,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "703"
      }
    },
    {
      "id": "E8AA690B5C6D29E3C0708A738E6105057172C2DE13A2208C18162FD69B87C29E",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464441,
      "block": 54787,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "563"
      }
    },
    {
      "id": "942F4C718E8BDEB56A8F5BBE1FA795E9185F0CF021C5CA666FFF92C1D9643B43",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464357,
      "block": 54775,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "858"
      }
    },
    {
      "id": "BD692045F53344715F60E4C10E8239BC5CC63BA1020974E5128FE014C062D1A5",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464244,
      "block": 54759,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "422"
      }
    },
    {
      "id": "F9B01B403A839AABA5FC6EE70F8AAE371BBD1CA4FF95511EAFB15C0B21AA405F",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464181,
      "block": 54750,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "328"
      }
    },
    {
      "id": "1C385B77B61F9F86A678BE6E6697E25DDA02A1A19467663BE5B527CFA3B74971",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464133,
      "block": 54743,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "187"
      }
    },
    {
      "id": "5206A7418AA9B10F58FA71619955661BA65D0874560B775BAA6AE8827E684244",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464105,
      "block": 54739,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "704"
      }
    },
    {
      "id": "CCDE53CB07ED7737D9469DDD9F77F6EE4E26DD9C4B779AF7F38C45CBAF537332",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576464000,
      "block": 54724,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "483"
      }
    },
    {
      "id": "A2F7387A4DACB21783F6AD783941132E8A723A6332F7DD2D408B754DEDD5E1AA",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463944,
      "block": 54716,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "422"
      }
    },
    {
      "id": "1B42D164A8B0E57AF9631B9A7A01C3C4837609D0636D3080858981BC549FB38F",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463883,
      "block": 54707,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "328"
      }
    },
    {
      "id": "21883FE51BBE72CCD4CCEE0798855AF89C9FDC62DB760928334C204D2CF363B4",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463833,
      "block": 54700,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "328"
      }
    },
    {
      "id": "AB8BD99929F46724435C1A21B8C9E2A7753B31F804DA7FB07F7B22484E23BD15",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463784,
      "block": 54693,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "999"
      }
    },
    {
      "id": "041BEABFEF1EA9F20F97FF30AD568999A93E378A44F29AE1731F23B837D7F014",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463650,
      "block": 54674,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "657"
      }
    },
    {
      "id": "8332B3F709BEDD6E17AB2F87CD42E2D71113AA48CFFCC2012C242A310A5E68AB",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463552,
      "block": 54660,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "750"
      }
    },
    {
      "id": "E0929B970BA40C58AA37A43AF80FE1D689EC3A780F74921D090BADF484F8F892",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463440,
      "block": 54644,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "765"
      }
    },
    {
      "id": "2C242229F44F14318A4526116E6B40F0514E78D667D5D57D9D08870C134086E6",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463345,
      "block": 54630,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "422"
      }
    },
    {
      "id": "843D85F28CC07EF3610BE5A5E6929241AA2F37DE8BEE2EC946C757DFC6B9C0A5",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576463283,
      "block": 54621,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "2924"
      }
    },
    {
      "id": "082BA88EC055A7C343A353297EAC104CE87C659E0DDD84621C9AC3C284232800",
      "coin": 118,
      "from": "cosmos1y6yvdel7zys8x60gz9067fjpcpygsn62ae9x46",
      "to": "cosmosvaloper12w6tynmjzq4l8zdla3v4x0jt8lt4rcz5gk7zg2",
      "fee": "0",
      "date": 1576462863,
      "block": 54561,
      "status": "completed",
      "type": "any_action",
      "direction": "incoming",
      "memo": "复投",
      "metadata": {
        "coin": 118,
        "title": "Claim Rewards",
        "key": "stake_claim_rewards",
        "token_id": "",
        "name": "ATOM",
        "symbol": "ATOM",
        "decimals": 6,
        "value": "2692701"
      }
    }
  ],
  "status": true
}
```

### CODE REVIEW

**Comment by vikmeup** [comment]
```
"title": "Get Reward",
"key": "stake_delegate",
```

`stake_delegate` is `outgoing` transaction, it's not rewards

we need separate action: `stake_claim_rewards` that will be incoming transaction

**Comment by Pantani** [comment]
> ```
> "title": "Get Reward",
> "key": "stake_delegate",
> ```
> 
> `stake_delegate` is `outgoing` transaction, it's not rewards
> 
> we need separate action: `stake_claim_rewards` that will be incoming transaction

done!! the example is updated as well

File: pkg/blockatlas/tx.go [hunk 1]
```diff
@@ -28,23 +28,22 @@ const (
...
-
-	KeyTitlePlaceOrder  KeyTitle = "Place Order"
-	KeyTitleCancelOrder KeyTitle = "Cancel Order"
-)
-
-// Titles of AnyAction meta
-const (
-	AnyActionDelegation   = "Delegation"
-	AnyActionUndelegation = "Undelegation"
+	TxGetReward           TransactionType = "get_reward"
```
Comment (vikmeup) [inline pkg/blockatlas/tx.go]: `TxGetReward ` unused ?

File: pkg/blockatlas/tx.go [hunk 2]
```diff
@@ -28,23 +28,22 @@ const (
...
+	KeyMintToken         KeyType = "mint_token"
+	KeyApproveToken      KeyType = "approve_token"
+	KeyStakeDelegate     KeyType = "stake_delegate"
+	KeyStakeClaimRewards KeyType = "stake_claim_rewards"
+
+	KeyTitlePlaceOrder    KeyTitle = "Place Order"
+	KeyTitleCancelOrder   KeyTitle = "Cancel Order"
+	AnyActionDelegation   KeyTitle = "Delegation"
+	AnyActionUndelegation KeyTitle = "Undelegation"
+	AnyActionClaimRewards KeyTitle = "Get Rewards"
```
Comment (vikmeup) [inline pkg/blockatlas/tx.go]: `Get Rewards` => `Claim Rewards `

**Pull request merged**